site/content/en/docs/Guides/Client SDKs/_index.md

```diff
@@ -258,14 +258,9 @@ If `GameServer.Status.Players.Capacity` is set manually through the Kubernetes A
 
 #### Alpha().GetPlayerCount()
 
-This function returns if the playerID is currently connected to the GameServer. 
-This is always accurate from what has been set through this SDK,
-even if the value has yet to be updated on the GameServer status resource.
-
-{{< alert title="Note" color="info">}}
-If `GameServer.Status.Players.IDs` is set manually through the Kubernetes API, use SDK.GameServer() 
-or SDK.WatchGameServer() instead to retrieve the current player count.
-{{< /alert >}}
+This function retrieves the current player count. 
```

Comment:
Thanks for sending this through!

We should keep the note about "If `GameServer.Status.Players.IDs` is set manually through the Kubernetes API..." because it is still relevant (and important too :smile:)